src/cltk/core/data_types.py

```diff
@@ -105,6 +106,25 @@ def __getattr__(self, item: str):
             raise AttributeError(item)
 
 
+@dataclass
+class Sentence:
```

Comment:
Could it become a parent of a `Verse` class?

Reply:
For your `Verse` class, Clement, I think it should be something new. (In theory, it could subclass from `Doc`, but still I would prefer that it stand alone.)

Reply:
I think we should think of Sentences as Verses as Views of a doc.
We will always have a Doc, that is a list of tokens, and those tokens can be viewed as a list of sentences, or a list of verses, but the sentence and view boundaries don't always align.

Which brings up and issue, we should probably allow Doc creation using line breaks that creates verse objects, e.g.
```doc = nlp("Odi et amo ... \n nescio....")```
Doc(.... verse_index=1, verse_index=2
etc...